lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -666,7 +665,9 @@ _SPIRV_OP(UDiv)
 _SPIRV_OP(SDiv)
 _SPIRV_OP(FDiv)
 _SPIRV_OP(SRem)
+_SPIRV_OP(SMod)
 _SPIRV_OP(FRem)
+_SPIRV_OP(FMod)
```

Comment:
Nit: it might be better to reorder this as "SRem, FRem, UMod, SMod, FMod" to as to preserve the pattern of listing the (signed, and then unsigned, if applicable) integer op followed by the corresponding floating point op, which occurs throughout this sequence of typedefs.

Reply:
I have followed the order of the `Op` enum values from `spirv.hpp`:
```
    OpSRem = 138,
    OpSMod = 139,
    OpFRem = 140,
    OpFMod = 141,
```
I'd argue sticking to the enum order makes more sense than inventing a different pattern.  Funny enough, the SPIR-V specification's enum values follow the pattern you're describing for most of the binary arithmetic ops.

Reply:
> I'd argue sticking to the enum order makes more sense than inventing a different pattern. Funny enough, the SPIR-V specification's enum values follow the pattern you're describing for most of the binary arithmetic ops.

Okay, that makes sense, although UMod would be out of place, in that case (although this is not new).